client/admin.go

```diff
@@ -0,0 +1,123 @@
+package client
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/p2p"
+)
+
+// AddPeer connects to the given nodeURL.
+func (c *client) AddPeer(ctx context.Context, nodeURL string) (bool, error) {
+	var r bool
+	// TODO: Result needs to be verified
+	// The response data type are bytes, but we cannot parse...
```

Comment:
Is it still a TODO item?
It seems we obtain the response data and return it as a `bool`

Reply:
We should remove it